src/reuse/comment.py

```diff
@@ -893,6 +893,7 @@ class XQueryCommentStyle(CommentStyle):
     "meson.build": PythonCommentStyle,
     "meson_options.txt": PythonCommentStyle,
     "pubspec.lock": UncommentableCommentStyle,
+    "poetry.lock": UncommentableCommentStyle,
     "pylintrc": PythonCommentStyle,
```

Comment:
Ordered alphabetically

```suggestion
    "meson_options.txt": PythonCommentStyle,
    "poetry.lock": UncommentableCommentStyle,
    "pubspec.lock": UncommentableCommentStyle,
    "pylintrc": PythonCommentStyle,
```

Reply:
Fixed this :)